src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return color.name();
     }
 }
```

Comment:
The class 'ColorSupplier' is using a new instance of 'Random' every time the 'getRandomColor' method is called. This is not efficient as you can reuse a single instance of 'Random'. Consider declaring 'Random' as a private final class field.